src/docs/product/sentry-basics/key-terms.mdx

```diff
@@ -0,0 +1,59 @@
+
+- **alerts** - Alerts let you know about problems with your code in real-time by sending you notifications when certain alert rule conditions are met. There are several types of alerts available with customizable thresholds and integrations.
+
+## Key Features
+
+Each of these key features in [sentry.io](https://sentry.io) is represented by a page (or set of pages) in the application.
+
+- **Projects** - Lists the projects of which you're a member, by team, and provides you with a high-level overview of your projects. From here, you can go to the **Project Details** page of each project for a more granular view. Learn more in the full [Projects documentation](/product/projects/).
+
+- **Issues** - Displays information about grouped errors in your application. From here, you can go to the **Issue Details** page for a more granular view of any issue. Learn more in the full [Issues documentation](/product/issues/).
+
+- **Performance** - The main view in [sentry.io](http://sentry.io) where you can search or browse for transaction data. The page displays graphs that visualize transactions or trends, as well as a table where you can view relevant transactions and drill down to more information about them. Learn more in the full [Performance documentation](/product/performance/).
+
+- **Releases** - Provides a high-level view of each release version, the associated project, the adoption stage of each release, the authors of each commit, the percentage of crash-free users, and the percentage of crash-free sessions. Learn more in the full [Releases documentation](/product/releases/).
+
+- **Release Health** - Provides insight into the impact of crashes and bugs as it relates to your user's experience and reveals trends with each new issue. You can navigate to release health data either from the **Issue Details** page, by selecting release ID listed under "Last Seen", or by going to the **Releases** page. Learn more in the full [Release Health documentation](/product/releases/health/).
+
+- **Alerts** - Where you can create new alert rules and manage existing ones. Learn more in the full [Alerts documentation](/product/alerts/).
+
+- **Discover** - Allows you to query events across environments, providing you with visibility into your errors and transactions and unlocking insights into the health of your entire system. The **Discover** page visualizes the results of queries. Learn more in the full [Discover documentation](/product/discover-queries/).
+
+- **Dashboards** - Provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more **Discover** queries. Learn more in the full [Dashboards documentation](/product/dashboards/).
```

Comment:
We also have support for querying Issues data, which is different from Discover queries. We don't have a big blurb on it, just this little part here
https://docs.sentry.io/product/dashboards/custom-dashboards/#data-set-selection

Reply:
```suggestion
- **Dashboards** - Provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more **Discover**/**Issues** queries. Learn more in the full [Dashboards documentation](/product/dashboards/).
```

@edwardgou-sentry is making improvements to our Data Set Selection docs in #4905 so hopefully the differences become more clear

Reply:
Makes sense. I've just replaced the / with the word "or" instead.